packages/i18n/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@web-stories-wp/i18n",
+  "private": true,
+  "engines": {
+    "node": ">= 15",
+    "npm": "~7"
+  },
+  "exports": {
+    "default": "./src/index.js"
+  },
+  "main": "./src/index.js",
+  "type": "module",
+  "dependencies": {
+    "@wordpress/i18n": "^3.17.0",
+    "react": "^16.14.0",
+    "react-dom": "^16.14.0",
```

Comment:
This is only a dev dependency used in tests

Reply:
Fixed